Marlin/temperature.cpp

```diff
@@ -1353,12 +1466,12 @@ ISR(TIMER0_COMPB_vect)
 
 #if HEATER_BED_PIN > -1
   static unsigned char soft_pwm_b;
-#ifdef SLOW_PWM_HEATERS
-  static unsigned char state_heater_b = 0;
-  static unsigned char state_timer_heater_b = 0;
-#endif 
+  #ifdef SLOW_PWM_HEATERS
```

Comment:
They are indented because they are within another #if / #endif. I think it helps the readability.